packages/frontend/src/components/SimpleChart/index.tsx

```diff
@@ -16,6 +16,7 @@ const flipXFromChartType = (chartType: DBChartTypes) => {
         case DBChartTypes.COLUMN:
         case DBChartTypes.LINE:
         case DBChartTypes.SCATTER:
+        case DBChartTypes.BIG_NUMBER:
```

Comment:
@nathaliabj you shouldn't need this part

Reply:
I suppose we need to exclude this type from the argument so typescript doesn't complain about the switch not handling that type.